docker-compose.yml

```diff
@@ -24,13 +24,22 @@ services:
     <<: *app
     image: beabee/beabee-cron-app:release
 
-  img_upload_app:
-    image: beabee/pictshare:release
+  minio:
+    image: beabee/beabee-minio:release
     restart: always
-    environment:
-      URL: ${BEABEE_AUDIENCE}/uploads/
     volumes:
-      - ./data/uploads:/var/www/data
+      - minio_data:/data
+    healthcheck:
```

Comment:
Should be in the image build step (as mentioned in https://github.com/beabee-communityrm/monorepo/pull/163)